cli.js

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env node
+'use strict'
+
+const fs = require('fs')
+const path = require('path')
+const process = require('process')
+
+const cli = require('cli')
+const mkdirp = require('mkdirp')
+
+const { transformFile } = require('./index')
+
+const CWD = process.cwd()
+
+const extensions = ['js', 'es', 'es6']
+
+cli.parse({
+  in: ['i', 'The file or directory you would like to transform (required)', 'string'],
+  out: ['o', 'Where to write output', 'string'],
+  sourceMaps: ['s', 'Output sourceMap']
+})
+
+cli.main(function main (args, options) {
+  if (!options.in) {
+    throw new Error('Must provide an input file or directory')
+  }
+
+  if (isDirectory(options.in)) {
+    if (!options.out) {
+      throw new Error('Must provide an output directory when transforming a directory')
+    }
+
+    return transformDirectory(options.in, options.out, options.sourceMaps)
+  }
+
+  const { code, map } = transformFile(options.in) // TODO: add options
```

Comment:
i think destructuring assignment is only available in node v6... we may want to support older versions?